app/src/assets/css/Selectable.css

```diff
@@ -10,6 +10,9 @@
   flex-direction: column;
   border-radius: 2px;
   transition: 0.25s border-color;
+  user-select: none;
+  cursor: pointer;
+  cursor: hand;
```

Comment:
Yeah, lots of people use pointer on things that aren't hyperlinks. Materialize seems to use pointer on all buttons. 

If the problem is that the element doesn't seem clickable then a pointer is not a good solution imo. Devices like mobile will never show the pointer anyway.